controller/pinctrl.c

```diff
@@ -2038,6 +2038,11 @@ pinctrl_handle_put_dhcp_opts(
     switch (*in_dhcp_msg_type) {
     case DHCP_MSG_DISCOVER:
         msg_type = DHCP_MSG_OFFER;
+        if (in_flow->nw_dst != htonl(0xffffffff)){
```

Comment:
It's probably nicer to use `htonl(INADDR_BROADCAST)`

Reply:
done